osu.Game/Overlays/Chat/ChannelControl/ControlItemClose.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics.Containers;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Overlays.Chat.ChannelControl
+{
+    public class ControlItemClose : OsuClickableContainer
+    {
+        private readonly SpriteIcon icon;
+
+        public ControlItemClose()
+        {
+            Alpha = 0f;
+            Size = new Vector2(20);
+            Child = icon = new SpriteIcon
+            {
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+                Scale = new Vector2(0.75f),
+                Icon = FontAwesome.Solid.TimesCircle,
+                RelativeSizeAxes = Axes.Both,
+            };
+        }
+
+        protected override bool OnMouseDown(MouseDownEvent e)
+        {
+            icon.ScaleTo(0.5f, 1000, Easing.OutQuint);
+            return base.OnMouseDown(e);
+        }
+
+        protected override void OnMouseUp(MouseUpEvent e)
+        {
+            icon.ScaleTo(0.75f, 1000, Easing.OutElastic);
+            base.OnMouseUp(e);
+        }
+
+        protected override bool OnHover(HoverEvent e)
+        {
+            icon.FadeColour(Color4.Red, 200, Easing.OutQuint);
```

Comment:
This red doesn't look great, too saturated in comparison with everything else. Would recommend trying `OsuColour.Red1` or similar.